org.kartkrew.RingRacers.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: org.kartkrew.RingRacers
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: ringracers.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --device=all
+  - --persist=.ringracers
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
+  - --filesystem=xdg-run/discord-ipc-0
+
+modules:
+  - name: ringracers
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://github.com/KartKrewDev/RingRacers/archive/refs/tags/v2.0.tar.gz
+        sha256: 55c51cf82104013d22935ec675d55c0aa21bba1f3ce3ad0903a33d96c35b23f9
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_C_FLAGS=-g1 -O3
+      - -DCMAKE_CXX_FLAGS=-g1 -O3 -fpermissive
+      - -DSRB2_CONFIG_DEV_BUILD=OFF
+      - -DSRB2_CONFIG_ENABLE_WEBM_MOVIES=OFF
+      - -DSRB2_SDL2_EXE_NAME=ringracers
+    post-install:
+      - mkdir -p $FLATPAK_DEST/bin
+      - mv $FLATPAK_DEST/ringracers $FLATPAK_DEST/bin/ringracers
+      - rm -f $FLATPAK_DEST/lib/libacsvm.a
+
+  - name: ringracers-meta
+    buildsystem: simple
+    build-commands:
+      - install -Dm 755 -t $FLATPAK_DEST/bin ringracers.sh
+      - install -Dm 644 $FLATPAK_ID.desktop -t $FLATPAK_DEST/share/applications
+      - install -Dm 644 $FLATPAK_ID.metainfo.xml -t $FLATPAK_DEST/share/metainfo
+      - install -Dm 644 $FLATPAK_ID.64.png $FLATPAK_DEST/share/icons/hicolor/64x64/apps/org.kartkrew.RingRacers.png
+    sources:
+      - type: file
+        path: ringracers.sh
+      - type: file
+        path: org.kartkrew.RingRacers.desktop
+      - type: file
+        path: org.kartkrew.RingRacers.metainfo.xml
+      - type: file
+        path: org.kartkrew.RingRacers.64.png
+
+  - name: ringracers-extra
+    buildsystem: simple
+    build-commands:
+      - install -t $FLATPAK_DEST/bin apply_extra
+    sources:
+      - type: extra-data
```

Comment:
why making this extra data. Aren't the assets redistributable?

Reply:
Switched away from extra data